### PR TITLE
Feature/embeddable

### PR DESCRIPTION
**Pull request by borysyuk**

### CODE REVIEW

File: app/front/scripts/services/navigation.js [hunk 1]
```diff
@@ -28,7 +28,7 @@
           filterList.push(key+'|'+value);
         });
 
-        var embed = (state.isEmbeded)? '/embed': '';
+        var embed = (state.isEmbedded)? '/embed': '';
```
Comment (pwalsh) [inline app/front/scripts/services/navigation.js]: @borysyuk wouldn't it be easier to pass a var from the server to definitively say that the state is embedded? That way, it does not rely on the url path, which is fragile/could change. etc.

**Comment by pwalsh** [comment]
@borysyuk just one comment here.

File: app/front/scripts/services/navigation.js (hunk 1 above)
Comment (borysyuk) [inline app/front/scripts/services/navigation.js]: @pwalsh I have seen new pull request from Adam  about base url and reletive urls. so, your way is better. I will take Adam's changes and will show/hide header and footer on backend.

**Comment by pwalsh** [comment]
@borysyuk great. Approved.

**Pull request merged**